### PR TITLE
[improvement](streamload) QeProcessorImpl#registerQuery should be removed

### PR DESCRIPTION
**Pull request by zzzzzzzs**
## Proposed changes
`QeProcessorImpl#registerQuery` should be removed 

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zzzzzzzs** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java [hunk 1]
```diff
@@ -2075,6 +2075,8 @@ private void httpStreamPutImpl(TStreamLoadPutRequest request, TStreamLoadPutResu
         } catch (Throwable e) {
             LOG.warn("exec sql error catch unknown result.", e);
             throw new UserException("exec sql error catch unknown result." + e);
+        } finally {
+            QeProcessorImpl.INSTANCE.unregisterQuery(ctx.queryId());
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java:2079]: why do we need register ctx to qeprocessorimpl?

**Comment by zzzzzzzs** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]
LGTM

**Review by zclllyybb — APPROVED** [review]

**Comment by yiguolei** [comment]
run buildall

**Comment by yiguolei** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by englefly — APPROVED** [review]

**Pull request merged**